pkgs/development/em-modules/generic/default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs, lib, emscripten, python2 }:
+{ pkgs, lib, emscripten, python }:
```

Comment:
why was this changed?

Reply:
That's a good question @Artturin. `git blame` places the use of python2 with 41faf39, which says that it also hasn't been tested with python3 yet. Given that `emscripten` has required python3 [for a while](https://github.com/emscripten-core/emscripten/issues/7198), I figured it'd be a reasonable time to update this too. Happy to switch it back to make the PR more conservative, but I'm also not sure what role python2 would play anymore in this stdenv.

Reply:
`python` is an alias to `python2` so thats why im asking

if it should be python3 then it should be `python3`

Reply:
Many apologies, I did not know that! Force-pushed to replace with python3. Build still works for me for the three non-broken packages.